package.json

```diff
@@ -24,6 +24,7 @@
   },
   "dependencies": {
     "@big-whale-labs/constants": "^0.1.22",
+    "@big-whale-labs/frontend-utils": "^0.0.6",
```

Comment:
For some reason, toast doesn't appear 🤔 

https://user-images.githubusercontent.com/49658988/187412181-38c66076-c407-4c3c-a116-671941918554.mp4

Reply:
Fixed it, `react-toastify` wasn't bundled properly in `frontend-utils`. Ty for catch